src/Illuminate/Http/Request.php

```diff
@@ -788,11 +780,11 @@ public function offsetUnset($offset)
 	 */
 	public function __get($key)
 	{
-		$input = $this->input();
+		$all = $this->all();
 
-		if (array_key_exists($key, $input))
+		if (array_key_exists($key, $all))
 		{
-			return $this->input($key);
+			return $this->all($key);
```

Comment:
`all()` doesn't take any arguments, also a function call is really unnecessary since you already have the array.
This should work:

    return $all[$key]